default-e2e-test-data/testcases/release-operator-e2e.json

```diff
@@ -4,5 +4,12 @@
   "desc": "helmrelease install test: guestbook deploy",
   "urls": ["https://raw.githubusercontent.com/open-cluster-management/multicloud-operators-subscription-release/e2e-test-setup/examples/test-guestbook010.yaml"],
   "target_cluster": "hub"
+},
+
+{
+  "test_id": "release-003",
+  "desc": "helmrelease with secret install test: guestbook deploy",
+  "url": ["https://raw.githubusercontent.com/pezhang/helm-release-API-test/gh-pages/test-guestbook010-secret.yaml"],
```

Comment:
over here, it's `urls` instead of `url`, the key for your URL

Reply:
@ianzhang366 I have fixed it. Thanks.